experimental/vlan-networks.md

```diff
@@ -210,44 +210,44 @@ Ipvlan L3 mode drops all broadcast and multicast traffic. This reason alone make
 - Unlike ipvlan l2 modes, different subnets/networks can ping one another as long as they share the same parent interface `-o parent=`.
 
 ```
-ip a show eth0
+$$ ip a show eth0
```

Comment:
There are two '$' here because for code the [Documentation style and grammar conventions](https://docs.docker.com/opensource/doc-style/#code-and-ui-text-styling) says:

- Start typed commands with $  (dollar space), so that they are easily differentiated from program output.
- In-container shell commands, begin with $$  (dollar dollar space).